Lib/fontTools/ttLib/tables/_c_m_a_p.py

```diff
@@ -15,6 +15,19 @@
 log = logging.getLogger(__name__)
 
 
+def _make_map(font, chars, gids):
+	assert len(chars) == len(gids)
+	cmap = {}
+	lenCmap = len(gids)
+	glyphOrder = font.getGlyphOrder()
+	try:
+		names = list(map(operator.getitem, [glyphOrder]*lenCmap, gids ))
+	except IndexError:
+		getGlyphName = font.getGlyphName
+		names = list(map(getGlyphName, gids ))
+	list(map(operator.setitem, [cmap]*lenCmap, chars, names))
```

Comment:
Building a list and just discarding is wasteful. Since nothing happens with `cmap` in the meantime, the following does the same, but (to me) more readable, and def. more Pythonic:
`cmap = {char: name for char, name in zip(chars, names)}`